tests/foreman/api/test_user.py

```diff
@@ -44,14 +44,15 @@
 from robottelo.helpers import read_data_file
 
 
+@pytest.fixture(scope='module')
+def create_user(self):
+    """Create an user"""
```

Comment:
```suggestion
    """Create a user"""
```

Reply:
At least Stephen didn't catch me doing this